CRM/Mailing/Tokens.php

```diff
@@ -72,7 +72,7 @@ public function checkActive(\Civi\Token\TokenProcessor $processor) {
    * @return array
    * @throws \Exception
    */
-  public function prefetch(\Civi\Token\Event\TokenValueEvent $e) {
+  public function prefetch(\Civi\Token\Event\TokenValueEvent $e, $activeTokens) {
```

Comment:
Docblock? As someone browsing the code, it'd help to have an example of the data that will be passed in here.